klayout/drc/rule_decks/esd.drc

```diff
@@ -0,0 +1,120 @@
+  logger.info('Executing rule ESD.3b')
+  esd3b_l1 = esd.not_outside(pcomp)
+  esd3b_l1.output('ESD.3b', 'ESD.3b : Min/max space to a butted PCOMP: 0um')
+  esd3b_l1.forget
+
+  # Rule ESD.4a: Extension beyond NCOMP is 0.24µm.
+  logger.info('Executing rule ESD.4a')
+  esd4a_l1 = ncomp.edges.enclosed(esd_edges.not_interacting(pcomp), 0.24.um, euclidian)
+  esd4a_l1.output('ESD.4a', 'ESD.4a : Extension beyond NCOMP: 0.24µm')
+  esd4a_l1.forget
+
+  # Rule ESD.4b: Minimum overlap of an ESD implant edge to a COMP is 0.45µm.
+  logger.info('Executing rule ESD.4b')
+  esd4b_l1 = esd.overlap(comp, 0.45.um, euclidian)
+  esd4b_l1.output('ESD.4b', 'ESD.4b : Minimum overlap of an ESD implant edge to a COMP. : 0.45µm')
+  esd4b_l1.forget
+
+  # Rule ESD.5a: Minimum ESD area (um2) is 0.49µm².
+  logger.info('Executing rule ESD.5a')
+  esd5a_l1 = esd.with_area(nil, 0.49.um)
+  esd5a_l1.output('ESD.5a', 'ESD.5a : Minimum ESD area (um2): 0.49µm²')
+  esd5a_l1.forget
+
+  # Rule ESD.5b: Minimum field area enclosed by ESD implant (um2) is 0.49µm².
+  logger.info('Executing rule ESD.5b')
+  esd5b_l1 = esd.holes.with_area(nil, 0.49.um)
+  esd5b_l1.output('ESD.5b', 'ESD.5b : Minimum field area enclosed by ESD implant (um2): 0.49µm²')
+  esd5b_l1.forget
+
+  # Rule ESD.6: Extension perpendicular to Poly2 gate is 0.45µm.
+  logger.info('Executing rule ESD.6')
+  esd6_l1 = esd_edges.enclosing(poly2.edges.interacting(tgate_edges), 0.45.um, projection)
+  esd6_l1.output('ESD.6', 'ESD.6 : Extension perpendicular to Poly2 gate: 0.45µm')
+  esd6_l1.forget
+  esd_edges.forget
+
+  # Rule ESD.7: No ESD implant inside PCOMP.
+  logger.info('Executing rule ESD.7')
+  esd7_l1 = esd.and(pcomp)
+  esd7_l1.output('ESD.7', 'ESD.7 : No ESD implant inside PCOMP.')
+  esd7_l1.forget
+
+  # Rule ESD.8: Minimum space to Nplus/Pplus is 0.3µm.
+  logger.info('Executing rule ESD.8')
+  esd8_l1 = esd.separation(nplus.join(pplus), 0.3.um)
+  esd8_l1.output('ESD.8', 'ESD.8 : Minimum space to Nplus/Pplus: 0.3µm')
+  esd8_l1.forget
+
+  # Rule ESD.pl: Minimum gate length of 5V/6V gate NMOS is 0.8µm.
+  logger.info('Executing rule ESD.pl')
+  esdpl_l1 = poly2.and(dualgate).interacting(esd).edges.and(tgate_edges)
+  esdpl_l  = esdpl_l1.width(0.8.um, euclidian)
+  esdpl_l.output('ESD.pl', 'ESD.pl : Minimum gate length of 5V/6V gate NMOS: 0.8µm')
+  esdpl_l1.forget
+  esdpl_l1.forget
+  tgate_edges.forget
+
+  # Rule ESD.9: ESD implant layer must be overlapped by Dualgate layer
+  ## (as ESD implant option is only for 5V/6V devices).
+  logger.info('Executing rule ESD.9')
+  esd9_l1 = esd.not_overlapping(dualgate)
+  esd9_l1.output('ESD.9', 'ESD.9 : ESD implant layer must be overlapped by Dualgate layer
+                  (as ESD implant option is only for 5V/6V devices).')
+  esd9_l1.forget
+
+  # Rule ESD.10: LVS_IO shall be drawn covering I/O MOS active area by minimum overlap is 0um.
+  logger.info('Executing rule ESD.10')
+  esd10_l1 = comp.and(esd).not_outside(lvs_io).not(lvs_io)
```

Comment:
would `.and(lvs_io)` also work here? does `I/O MOS active area` wording of the rule correspond to `comp.and(esd)`?